src/Monomer/Graphics/Text.hs

```diff
@@ -53,25 +53,26 @@ calcTextSize
   -> Text          -- ^ The text to calculate.
   -> Size          -- ^ The calculated size.
 calcTextSize fontMgr style !text = size where
-  size = calcTextSize_ fontMgr style SingleLine KeepSpaces Nothing Nothing text
+  size = calcTextSize_ fontMgr style SingleLine KeepSpaces OnSpaces Nothing Nothing text
```

Comment:
I think in this case it would be worth trying to use the new attribute from `StyleState`.

There are a few utilities in `StyleUtil.hs` for accessing `StyleState` fields (`styleTextAlignH`, `styleTextAlignV`, etc). Adding one for `styleTextLineBreak` would simplify retrieving this value (or the default if not available).